netflix-roulette/src/utils/constants.ts

```diff
@@ -0,0 +1,124 @@
+export enum GenreType {
+    DOCUMENTARY = "Documentary",
+    COMEDY = "Comedy",
+    HORROR = "Horror",
+    CRIME = "Crime",
+    ALL = "All",
+}
+
+export const GENRES = [GenreType.ALL, GenreType.DOCUMENTARY, GenreType.COMEDY, GenreType.HORROR, GenreType.CRIME];
```

Comment:
Just for your notice, you may achieve the same with:
```ts
export const GENRES = Object.values(GenreType)
```

Reply:
Thanks, updated